common/lib/common-definitions/package.json

```diff
@@ -50,6 +50,7 @@
     "@fluidframework/eslint-config-fluid": "^0.28.2000",
     "@microsoft/api-extractor": "^7.22.2",
     "@rushstack/eslint-config": "^2.5.1",
+    "buffer": "^6.0.3",
```

Comment:
I'm a little bit confused by this. You are saying "all server packages", but this is not server package. Moreover, definition packages should not use Buffer (that would be a bug that needs to be addressed by removing such dependency) and I can't find any reference in two packages I spot checked.
Maybe "these libraries" mean something else?
Would love to get clarity.